### ISSUE TITLE
CSS filters don't seem to work on 0.12.6

### ISSUE THREAD

**Issue by nvma** [wkhtmltopdf/wkhtmltopdf#4905]
**wkhtmltopdf version(s) affected**: 0.12.6

**OS information**
Ubuntu 20.04.1 LTS

**Description**
CSS filters (e.g. `-webkit-filter: blur(10px);`) don't work.

**How to reproduce**
```
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <style>
    .blurred {
      -webkit-filter: blur(20px);
      filter: blur(20px);
    }
  </style>
  <body>
    <img class="blurred" src="https://placekitten.com/200/287" />
  </body>
</html>

```
**Expected behavior**
Image is blurred.

**Possible Solution**  
Filters used to work on `0.12.5`. Maybe regression?

**Comment by ghnp5** [comment]
Blurring is a critical feature for me, but I had to go through a very overengineered solution just to be able to do blurring. And, I'm currently looking into other options because of this.

I saw this, and straight away downgraded to test 0.12.5, to see if the filters work.
But no... doesn't seem to work for me (tried your code above).

Are you sure it ever worked with wkhtmltopdf ??

**Comment by nvma** [comment]
Yes, blurring is working currently in production, but it still has some serious quirks with fonts rendering and SVG support. You should probably go with the puppeteer, it supports all of the features out of the box and probably has implementation for the language you use.

**Comment by ghnp5** [comment]
Thanks! I was just trying now with "weasyprint" and "xhtml2pdf", and both of these don't support "filter: blur()" either !
I'll give it a try with "puppeteer"...
Have a good day.

**Comment by ghnp5** [comment]
Puppeteer supports CSS3, which is absolutely great! Was able to create a perfect PDF.

However, now I was trying a *huge* HTML file (which worked well on wkhtmltopdf, except for the blur), and Chrome fills up the whole RAM and then crahes!!! :(

I guess I'll have to work on splitting into smaller PDFs and then merge them, or so :)

But yeah, I'm happy with Puppeteer!
